src/utils.js

```diff
@@ -738,10 +738,10 @@ export const getLatestHighestCpmBid = getHighestCpmCallback('responseTimestamp',
 
 function getHighestCpmCallback(useTieBreakerProperty, tieBreakerCallback) {
   return (previous, current) => {
-    if (previous.cpm === current.cpm) {
+    if (parseFloat(previous.cpm) === parseFloat(current.cpm)) {
```

Comment:
This is not necessary since https://github.com/prebid/Prebid.js/pull/9273, as targeting happens later and cpm here would always be a number. 

However, it may make sense to modify the [currency module](https://github.com/prebid/Prebid.js/blob/bc68d77cf85409c5f9f2c184a7e10bef2169d348/modules/currency.js#L237) so that it doesn't set a string in the first place. I'm not sure of the history there - @robertrmartinez, is there any use in having a fixed precision string that only exist between the currency logic and the end of the auction?

Reply:
Oh would you look at that! Already fixed! We at magnite are a few versions behind of course.

I cannot imagine why it would matter to contain exact precision @dgirardi 

But maybe it is safe to just parse after toFixed here? Not sure why it is needed to be to fixed 4 though.

Reply:
@robertrmartinez not sure i follow; your comment, is your recommendation to merge this even though the original issue is fixed?